packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -470,6 +477,11 @@ export class Cluster extends ClusterBase {
    */
   protected parameterGroup?: IClusterParameterGroup;
 
+  /**
+   * Whether the cluster will be rebooted when changes to the cluster's parameter group that require a restart to apply.
+   */
+  protected rebootForParameterChangesEnabled?: boolean;
```

Comment:
I don't think we need this prop, can we remove it? It seems redundant with `rebootForParameterChanges`?

Reply:
Related to my other comment `rebootForParameterChanges` is the class property that is needed so we don't create duplicate custom resources with `enableRebootForParameterChanges()`. I thought it would be clearer to have this property versus making the `Provider` and `CustomResource` class properties

Reply:
Please add to this docstring that this is only used to guard against repeated invocations of `enableRebootForParameterChanges()`